scripts/pre-commit.sh

```diff
@@ -68,6 +68,13 @@ then
 	say_done
 fi
 
+if test "$(git config --bool vimwiki.allhtml || echo false)" = true
```

Comment:
Thanks @jfishe! All of this looks fantastic! I don't normally use `VimwikiAll2HTML` - would it make sense to move this block after `vimwiki.generatetaglinks` so the updated tags page would be generated? Also, please add a small bit of text to the top of the script to identify the new variable.

Reply:
Agreed. Thanks for the feedback. https://github.com/sstallion/vimwiki-cli/pull/13/commits/5121f3468501b67dbcec10246ba4e6683fcc4619 should cover the changes.

Reply:
Always! Everything looks great to me - I'll get this merged and a new release out shortly. Thanks again for your contribution!